planning/src/parsing/pddl.rs

```diff
@@ -273,6 +276,24 @@ impl Display for Action {
         write!(f, ")")
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct DurativeAction {
+    pub name: Sym,
+    pub args: Vec<TypedSymbol>,
+    pub duration: Vec<SExpr>,
+    pub conditions: Vec<SExpr>,
```

Comment:
@arbimo Should we add `cargo fmt` and `cargo clippy` as prehook commands?

Reply:
> @arbimo Could we add `cargo fmt` and `cargo clippy` as prehook commands?

I am usually not a huge fan of prehook because they can be constraining. We only want to enforce the no-warnings and formated code when merging into master (and typically only on the last commit of the branch). So the CI for testing them seem the most appropriate solution. Note that hooks can be added locally if that's more inline with your workflow.